velox/type/tz/TimeZoneMap.cpp

```diff
@@ -26,23 +26,56 @@
 namespace facebook::velox::tz {
 
 // Defined in TimeZoneDatabase.cpp
-extern const std::unordered_map<int64_t, std::string>& getTimeZoneDB();
+extern const std::vector<std::pair<int16_t, std::string>>& getTimeZoneEntries();
+
+// TODO: The string will be moved to TimeZone in the next PR.
+using TTimeZoneDatabase = std::vector<std::unique_ptr<std::string>>;
```

Comment:
Why do you use `unique_pr<string>`?

Reply:
In the next PR I'll move this to unique_ptr<TimeZone>, but I worried it would make this PR too large to review, so this seemed like a good compromise. For now unique_ptr<string> because I need to represent that some entries are empty (nullptr).